test/evergreen.yml

```diff
@@ -3110,12 +3110,11 @@ tasks:
     commands:
       - func: "get project"
       - func: "compile wiredtiger"
-      # FIXME-WT-9770
-      # - func: "format test script"
-      #   vars:
-      #     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
-      #     # run for 10 minutes.
-      #     format_test_script_args: -t 10 rows=10000 ops=50000
+      - func: "format test script"
+        vars:
+          smp_command: -j $(grep -c ^processor /proc/cpuinfo)
```

Comment:
We can drop `smp_command` here. These days it's only in the buildvariant.

Reply:
Removed in [fd4edd6](https://github.com/wiredtiger/wiredtiger/pull/8712/commits/fd4edd6dfbf59f54136d60a3f2098a7eed6d6ae7).